backend/src/main/java/wooteco/retrospective/config/LoginInterceptor.java

```diff
@@ -0,0 +1,40 @@
+package wooteco.retrospective.config;
+
+import org.springframework.http.HttpMethod;
+import org.springframework.web.servlet.HandlerInterceptor;
+import wooteco.retrospective.exception.AuthorizationException;
+import wooteco.retrospective.infrastructure.auth.AuthorizationExtractor;
+import wooteco.retrospective.infrastructure.auth.JwtTokenProvider;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class LoginInterceptor implements HandlerInterceptor {
+
+    private JwtTokenProvider jwtTokenProvider;
+
+    public LoginInterceptor(JwtTokenProvider jwtTokenProvider) {
+        this.jwtTokenProvider = jwtTokenProvider;
+    }
+
+    @Override
+    public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler) {
+
+        if (request.getMethod().equals(HttpMethod.OPTIONS.name())) {
+            return true;
+        }
+
+        String accessToken = request.getHeader("Authorization");
+
+        if (accessToken == null) {
+            throw new AuthorizationException("유효하지 않은 토큰입니다.");
+        }
```

Comment:
```suggestion
        Optional.ofNullable(request.getHeader(HttpHeaders.AUTHORIZATION))
                .orElseThrow(() -> new AuthorizationException("유효하지 않은 토큰입니다."));
```

프로덕션에서 null을 이용하는 로직은 정말 안좋다고 생각해요. 이런식의 리팩토링을 진행하는게 어떨까요?:
또한 이 경우는 유효하지 않은 토큰이 아니라, 토큰이 존재하지 않는 에러 아닌가요?